src/pgen/parser/token.py

```diff
@@ -67,10 +67,12 @@
 AWAIT = 54
 ASYNC = 55
 # Extras hacked in
-COMMENT = 56
-NL = 57
-ERRORTOKEN = 58
-N_TOKENS = 59
+NOTEQUAL_PY2ONLY = 56
```

Comment:
I feel slightly queasy about adding a token and diverging from python on token numbers, but I haven't been able to figure out what a problem would be.